### PR TITLE
Clearify wording

### PR DESCRIPTION
**Pull request by ShalokShalom**
Which screenshot? :smiley:

### CODE REVIEW

**Comment by deansheather** [comment]
The screenshot is supposed to be the `` at the end of the line, but it looks like there's one too many `../`s in the URL.

**Comment by ShalokShalom** [comment]
How knows the new URL?

**Comment by deansheather** [comment]
Just remove one of the `../`s from the URL.

File: doc/self-hosted/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ It takes just a few minutes to get your own self-hosted server running. If you'v
 > NOTE: Be careful with your password as sharing it will grant those users access to your server's file system
 
 ### Things To Know
-- When you visit the IP for your code-server instance, you will be greeted with a page similar to the following screenshot. Code-server is using a self-signed SSL certificate for easy setup. In Chrome/Chromium, click **"Advanced"** then click **"proceed anyway"**. In Firefox, click **Advanced**, then **Add Exception**, then finally **Confirm Security Exception**.<img src ="../../assets/chrome_warning.png">
+- When you visit the IP for your code-server instance, you will be greeted with a screen similar to that one in the screenshot. Code-server is using a self-signed SSL certificate for easy setup. In Chrome/Chromium, click **"Advanced"** then click **"proceed anyway"**. In Firefox, click **Advanced**, then **Add Exception**, then finally **Confirm Security Exception**.<img src ="../assets/chrome_warning.png">
```
Comment (deansheather) [inline doc/self-hosted/index.md]: This wording is less clear and should be reverted to what it was originally.

Comment (deansheather) [reply doc/self-hosted/index.md]: This has not been resolved.

**Comment by deansheather** [comment]
@ShalokShalom Could you please make my requested changes so I can merge this?

**Comment by ShalokShalom** [comment]
Do it for yourself -.-

**Pull request closed without merging**